package.json

```diff
@@ -40,7 +40,8 @@
     "create-modules:native": "node ./scripts/release/createNativeModules.js",
     "create-modules:web": "node ./scripts/release/createWebModules.js",
     "version": "ts-node --project ./scripts/tsconfig.json ./scripts/release/BumpVersion.ts",
-    "validate-staged-widget-versions": "node scripts/validation/validate-versions-staged-files.js"
+    "validate-staged-widget-versions": "node scripts/validation/validate-versions-staged-files.js",
+    "data:widgets": "ts-node --project ./scripts/tsconfig.json ./scripts/data/PublishWidgetData.ts"
```

Comment:
I'm a bit hesitant about this naming this script `data:widgets` because we also have... data widgets module 😅  Maybe `update:widgets-data`?

Reply:
Good point. renamed it to `update:widgets-data`

Reply:
And now it's update:widgets-overview